core-api/src/controllers/permisson.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from "express";
+import { PermissonService } from "../services/permisson.service";
+import { HttpStatus } from "../utils/httpStatus";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import Messages from "../utils/messages";
+
+export class PermissonController {
```

Comment:
Permisson is wrong. The correct word is Permission. :smile:

Reply:
Sorry, misspelling.

Reply:
Done